wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -473,3 +473,43 @@ def avatar_url(user, size=50):
             return gravatar_url
 
     return static('wagtailadmin/images/default-user-avatar.png')
+
+
+@register.inclusion_tag("wagtailadmin/shared/icon.html", takes_context=False)
+def icon(name=None, class_name='icon', title=None):
+    """
+    Abstracts away the actual icon implementation.
+
+    Usage:
+        {% load wagtailadmin_tags %}
+        ...
+        {% icon name="cogs" classname="icon--red" title="Settings" %}
+
+    :param name: the icon name/id, required (string)
+    :param classname: default 'icon' (string)
+    :param title: accessible label intended for screen readers (string)
+    :return: Rendered template snippet (string)
+    """
+    if not name:
+        raise ValueError("You must supply an icon name")
+
+    return {
+        'name': name,
+        'class_name': class_name,
+        'title': title,
+    }
+
+
+@register.inclusion_tag("wagtailadmin/shared/icons.html")
+def icons():
+    icon_hooks = hooks.get_hooks('register_icons')
+    icons = sorted(itertools.chain.from_iterable(hook([]) for hook in icon_hooks))
+    return {'icons': icons}
+
+
+@register.filter
+def idify(val):
```

Comment:
This looks like it is no longer used and can be removed.

Reply:
Dropped